homeassistant/components/androidtv/__init__.py

```diff
@@ -33,16 +34,32 @@
     DEVICE_ANDROIDTV,
     DEVICE_FIRETV,
     DOMAIN,
+    PROP_ETHMAC,
     PROP_SERIALNO,
+    PROP_WIFIMAC,
     SIGNAL_CONFIG_ENTITY,
 )
 
+INVALID_MACS = ["ff:ff:ff:ff:ff:ff"]
```

Comment:
I'm no expert on MAC addresses. Is it common knowledge that this is an invalid MAC address?

Maybe preface the name with an underscore (`_INVALID_MACS`). And maybe just use a string (`_INVALID_MAC`), since there's only one, unless you expect there to be more in the future.

Reply:
I don't really think that this can exist. MAC address are provided by a dedicated entity and have some rules (for example the first part define the vendor). A mac address like that should mean "no mac address".

Reply:
> Maybe preface the name with an underscore (`_INVALID_MACS`)

Why?

> And maybe just use a string (`_INVALID_MAC`), since there's only one, unless you expect there to be more in the future

I used an array just in case, we will discover in the future other exceptions, it will be simpler.

Reply:
`"ff:ff:ff:ff:ff:ff"` cannot exist in the real world.

At set would be faster if you end up adding a lot of them since it doesn't have to do linear searching internally to lookup a value

Reply:
> Why?

The leading underscore makes it clear that it shouldn't be imported or used outside this module.

Reply:
> At set would be faster if you end up adding a lot of them since it doesn't have to do linear searching internally to lookup a value

you means convert to `{"ff:ff:ff:ff:ff:ff"}`

Reply:
Yes, note it only starts to matter when the item you want is at the end of the list or not in the list.  It usually easier to do it up front instead of changing all the typing later.